src/iogeneric/MimmoGeometry.cpp

```diff
@@ -382,6 +383,15 @@ MimmoGeometry::setClean(bool clean){
 	m_clean = clean;
 }
 
+/*!
```

Comment:
@edoardolombardi since it is used only to restore correctly dumped patches, can we rename it as setParallelRestore and specify into the doc we need it to reup correctly previously dumped patch? I dropped a note also for the twin xml Property "Parallel".